CI/unit_tests/losses/_test_proximal_policy_loss.py

```diff
@@ -1,4 +1,5 @@
-import numpy as np
```

Comment:
Please rename this to no have the leading underscore. This means it is being ignored by pytest.